backend/api/coworking/reservation.py

```diff
@@ -105,3 +106,10 @@ def get_total_hours_study_room_reservations(
 ) -> str:
     """Allows a user to know how many hours they have reserved in all study rooms (Excludes CSXL)."""
     return reservation_svc._get_total_time_user_reservations(subject)
+
+
+@api.get("/rooms/", tags=["Coworking"])
```

Comment:
Interesting question (will loop in @KrisJordan), do we want this in the rooms API file instead?

Reply:
Left as is since Kris has not answered! Also looping in @aarjavjain2002